### DISCUSSION TITLE
Should we delete https://pythonhosted.org/PyPDF2/ ?

### DISCUSSION THREAD

**Discussion by MartinThoma** [py-pdf/pypdf#682] [General]
The documentation on [pythonhosted.org ](https://pythonhosted.org/PyPDF2/) cannot be updated, but only be deleted permanently:

![image](https://user-images.githubusercontent.com/1658117/162560826-d54bd024-32aa-411d-893d-aa05803ebcda.png)

The reason is that this feature got deprecated from PyPI.

We do have the same documentation at https://pypdf2.readthedocs.io/en/latest/

I would vote for deleting the Pythonhosted documentation as it will grow more and more out of date. Maybe somebody could check beforehand if I forgot something to port to readthedocs?

**Comment by sekrause** [comment]
The [page about pdfcat](https://pythonhosted.org/PyPDF2/Easy%20Concatenation%20Script.html) is missing in the new documentation, is that still included in PyPDF2?

**Reply by MartinThoma** [reply]
There are two aspects to it: (1) What is on Github / within the PyPDF2 repository and (2) what is shipped with PyPDF2.

I've just looked up the latest release before my first one: https://pypi.org/project/PyPDF2/1.26.0/#files - to my surprise, I accidentially made a breaking change. `1.26.0` contained:

- Resources
- Sample_Code
- Scripts
- Tests

That was unexpected as I thought I had checked that before.

My approach now would be:

1. Add the pdfcat docs page
2. Make a patch release that contains it again
3. Remove it from PyPDF2 in 2.0.0 -- that one is up for discussion, but I would rather create a small package only for pdfcat / the other applications than mixing the two responsibilities.

**Reply by Butterfly** [reply]
> The documentation on [pythonhosted.org ](https://pythonhosted.org/PyPDF2/) cannot be updated, but only be deleted permanently:
> 
> I would vote for deleting the Pythonhosted documentation as it will grow more and more out of date. Maybe somebody could check beforehand if I forgot something to port to readthedocs?

Just a lurker out here, actually for a _very_ long time. The last few days' worth of activity has been phenomenal. As for this topic, words like "centralizing" and "avoiding scattered fragmentation" come to mind to make it as easy as possible for interested parties to track what you're doing.

That's a shame that piece of documentation can't be updated. It could have been used as a placeholder to advise where things have now migrated. Internet searches will hopefully have the project's back in that respect. I don't remember how I first found out about PyPDF2, but I wouldn't already be following unless some part of it seemed to be what others should strive to mime. Visibility, usability, accessibility are important keys there. :+1: 

N.B. *waving at Thomas* :laughing:
